web/app.js

```diff
@@ -1998,17 +1989,17 @@ function webViewerPageChanging(e) {
 
 var zoomDisabled = false, zoomDisabledTimeout;
 function handleMouseWheel(evt) {
-  var MOUSE_WHEEL_DELTA_FACTOR = 40;
-  var ticks = (evt.type === 'DOMMouseScroll') ? -evt.detail :
-              evt.wheelDelta / MOUSE_WHEEL_DELTA_FACTOR;
-  var direction = (ticks < 0) ? 'zoomOut' : 'zoomIn';
-
   var pdfViewer = PDFViewerApplication.pdfViewer;
   if (pdfViewer.isInPresentationMode) {
-    evt.preventDefault();
-    PDFViewerApplication.scrollPresentationMode(ticks *
-                                                MOUSE_WHEEL_DELTA_FACTOR);
-  } else if (evt.ctrlKey || evt.metaKey) {
+    return;
+  }
+
+  var delta = normalizeWheelEventDelta(evt);
+
+  var MOUSE_WHEEL_DELTA_PER_PAGE_SCALE = 3.0;
+  var ticks = delta * MOUSE_WHEEL_DELTA_PER_PAGE_SCALE;
```

Comment:
Nit: Perhaps a micro-optimization, but now that we're handling the `wheel` event in `pdf_presentation_mode.js`, I'm not sure that we need to compute the `ticks` here variable unconditionally. Hence I believe that you could move this code, i.e.
```javascript
  var delta = normalizeWheelEventDelta(evt);

  var MOUSE_WHEEL_DELTA_PER_PAGE_SCALE = 3.0;
  var ticks = delta * MOUSE_WHEEL_DELTA_PER_PAGE_SCALE;
```
inside of the `if (evt.ctrlKey || evt.metaKey) {` block below instead, to avoid computing `ticks` on normal scroll in the regular viewer.